docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2498 @@
 :issue: https://github.com/elastic/kibana/issues/
 :pull: https://github.com/elastic/kibana/pull/
 
-Review important information about the {kib} 8.0.0 releases.
+Review important information about the {kib} 8.5.0 release.
+
+* <<release-notes-8.5.0>>
+//* <<release-notes-8.4.3>>
+//* <<release-notes-8.4.2>>
+//* <<release-notes-8.4.1>>
+//* <<release-notes-8.4.0>>
+//* <<release-notes-8.3.3>>
+//* <<release-notes-8.3.2>>
+//* <<release-notes-8.3.1>>
+//* <<release-notes-8.3.0>>
+//* <<release-notes-8.2.3>>
+//* <<release-notes-8.2.2>>
+//* <<release-notes-8.2.1>>
+//* <<release-notes-8.2.0>>
+//* <<release-notes-8.1.3>>
+//* <<release-notes-8.1.2>>
+//* <<release-notes-8.1.1>>
+//* <<release-notes-8.1.0>>
+//* <<release-notes-8.0.1>>
+//* <<release-notes-8.0.0>>
+//* <<release-notes-8.0.0-rc2>>
+//* <<release-notes-8.0.0-rc1>>
+//* <<release-notes-8.0.0-beta1>>
+//* <<release-notes-8.0.0-alpha2>>
+//* <<release-notes-8.0.0-alpha1>>
 
-* <<release-notes-8.0.0>>
-* <<release-notes-8.0.0-rc2>>
-* <<release-notes-8.0.0-rc1>>
-* <<release-notes-8.0.0-beta1>>
-* <<release-notes-8.0.0-alpha2>>
-* <<release-notes-8.0.0-alpha1>>
+--
+[[release-notes-8.5.0]]
+== {kib} 8.5.0
+
+coming::[8.5.0]
+
+Review the following information about the {kib} 8.5.0 release.
+
+[float]
+[[breaking-changes-8.5.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 8.5.0, review the breaking changes, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+[discrete]
+[[breaking-141757]]
+.Updates bulk action API to return actionId instead of agents success for consistency
```

Comment:
```suggestion
.Updates bulk action API to return actionId instead of agent success
```

Reply:
I don't think we need to explain the reason. It makes this title grammatically awkward.